pyensembl/database.py

```diff
@@ -74,7 +74,7 @@ def _all_possible_indices(self, column_names):
         Create list of tuples containing all possible index groups
         we might want to create over tables in this database.
 
-        If a release is missing some column we want to index on,
+        If a genomic database is missing some column we want to index on,
```

Comment:
Yeah, I'm confused about this too. It's also confusing to call the whole bundle a `Genome` when really only the underlying whole genome sequence is, well, a genome. The transcript and protein FASTA files are derived concepts, as well as the GTF describing the coordinates of various genomic features. 

One thought:
*  "genomic feature annotations" == GTF

Other than that, I'm not sure.